### DISCUSSION TITLE
Edit text/image in a PDF page

### DISCUSSION THREAD

**Discussion by CrysK** [Stirling-Tools/Stirling-PDF#1086] [Q&A]
I have a vector pdf file. The text can be copied (as raw text). 
Is it possible to edit the text here and e.g. turn a "Here is a text" -> "Here is more text"?

Next to it is a picture on the page. Only this picture should be made smaller or moved. Is this possible?

Thank you very much!

**Comment by CrysK** [comment]
I want to have something like this:
![2024-04-30 17-25-42](https://github.com/Stirling-Tools/Stirling-PDF/assets/43782371/b558c0ec-b3d0-465d-ba26-5b61a8c280d0)

Here is the example:
[TestFile.pdf](https://github.com/Stirling-Tools/Stirling-PDF/files/15167156/TestFile.pdf)


Is this possible?

**Comment by CrysK** [comment]
It is sad that this is not possible and that no one can answer.

**Comment by ayushtiwariofficial** [comment]
No, As far as I've seen and explored Stirling-PDF this is not possible in Stirling in the current version.
